errorprone-checks/src/main/java/tech/pegasys/errorpronechecks/BannedMethod.java

```diff
@@ -34,12 +34,15 @@
 @BugPattern(
     name = "BannedMethod",
     summary = "Some methods should not be used, make sure that doesn't happen.",
-    severity = WARNING)
+    severity = WARNING,
+    linkType = LinkType.NONE)
 public class BannedMethod extends BugChecker implements MethodInvocationTreeMatcher {
 
   private static final ImmutableMap<Matcher<ExpressionTree>, String> BANNED_METHOD_LIST =
       ImmutableMap.of(
-          allOf(staticMethod().onClass("com.google.common.base.Objects").withAnyName()),
+          staticMethod().onClass("java.lang.System").named("currentTimeMillis"),
+          "Do not use System.currentTimeMillis(), use a java.time.Clock passed into a constructor or as a static method parameter.",
+          staticMethod().onClass("com.google.common.base.Objects").withAnyName(),
```

Comment:
Suggest adding a test for this new rule to `BannedMethodTest`

Reply:
It's already present.  [here](https://github.com/PegaSysEng/pantheon/pull/1747/files#diff-2614b8be9cbaff7fa1e2cb660f4bad9aR28) and [here](https://github.com/PegaSysEng/pantheon/pull/1747/files#diff-47d8ca838f0b31e45606b8d97a895f51R31)